widgetbook_for_widgetbook/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   cupertino_icons: ^1.0.2
   widgetbook_annotation: ^2.1.0
   widgetbook: ^2.4.1
+  flutter_bloc: ^8.1.1
```

Comment:
Hm.. This is a problem since it makes widgetbook usage more complicated.

Reply:
Fixed